README.md

```diff
@@ -58,6 +58,18 @@ Before submitting your pull request, you'll want to verify the changes in two wa
 
 All the tests for Rust exercises can be run from the top level of the repo with `_test/check-exercises.sh`. If you are on a Windows machine, there are additional [Windows-specific instructions](_test/WINDOWS_README.md) for running this.
 
+## On modifying the exercises' README
+
+Please note that the README of every exercise is formed using several templates, not necessary present on this repo:
```

Comment:
Let's expand to clarify:

```suggestion
Please note that the README of every exercise is formed using several templates, not all of which are necessary present on this repo. The most important of these:
```

Reply:
I you don't mind I will also decrease the new section header, so that it could become a subsection of the `Submitting a Pull Request` section